packages/whats-new/types.d.ts

```diff
@@ -1,12 +0,0 @@
-// TODO: Attempt removing after https://github.com/Automattic/wp-calypso/pull/78711
-// Fixed node_modules/@wordpress/components/build-types/dropdown-menu/types.d.ts(43,24): error TS2339: Property 'role' does not exist on type 'HTMLElement'.
-interface HTMLElement {
-	role: string;
```

Comment:
Hm, looks like we still need this. I'm pretty confused why adding it in whats-new fixes it, so I want to dig into the issue further.